alfalfa_worker/jobs/openstudio/create_run.py

```diff
@@ -0,0 +1,118 @@
+
+
+import json
+import os
+from subprocess import call
+
+from alfalfa_worker.jobs.openstudio import lib_dir
+from alfalfa_worker.lib.job import Job, JobExceptionInvalidModel
+from alfalfa_worker.lib.run import RunStatus
+from alfalfa_worker.lib.sim_type import SimType
+from alfalfa_worker.lib.tagutils import make_ids_unique, replace_site_id
+from alfalfa_worker.lib.utils import rel_symlink
+
+
+class CreateRun(Job):
+
+    def __init__(self, upload_id, model_name):
+        self.create_run_from_model(upload_id, model_name, SimType.OPENSTUDIO)
+
+    def exec(self):
+        self.set_run_status(RunStatus.PREPROCESSING)
+        osws = self.run.glob("**/*.osw")
+        if osws:
+            # there is only support for one osw at this time
+            submitted_osw_path = osws[0]
+            submitted_workflow_path = os.path.dirname(submitted_osw_path)
+        else:
+            raise JobExceptionInvalidModel("No .osw file found")
+
+        # locate the "default" workflow
+        default_workflow_path: str = os.path.join(lib_dir, 'workflow/workflow.osw')
+
+        # Merge the default workflow measures into the user submitted workflow
+        call(['openstudio', os.path.join(lib_dir, 'merge_osws.rb'), default_workflow_path, submitted_osw_path])
+
+        # run workflow
+        call(['openstudio', 'run', '-m', '-w', submitted_osw_path])
+
+        points_json_path = os.path.join(submitted_workflow_path, 'reports/haystack_report_haystack.json')
+        mapping_json_path = os.path.join(submitted_workflow_path, 'reports/haystack_report_mapping.json')
```

Comment:
```suggestion
        mapping_json_path = os.path.join(submitted_workflow_path, 'reports', 'haystack_report_mapping.json')
```

Would be nice to use `pathlib.Path`...